README.md

```diff
@@ -2,9 +2,15 @@
 
 Please see http://godoc.org/github.com/spacemonkeygo/openssl for more info
 
+### About
+
+This reposiroty is a fork of [spacemonkeygo OpenSSL](https://github.com/spacemonkeygo/openssl)
```

Comment:
they call themselves "[OpenSSL bindings for Go](https://github.com/spacemonkeygo/openssl#openssl-bindings-for-go)", taking into account that it originates from "[Space Monkey Go](https://github.com/spacemonkeygo)", maybe suggested rewording to:
```
This repository is a fork of [Space Monkey Go OpenSSL bindings](https://github.com/spacemonkeygo/openssl)
```
(also small misspell in `reposiroty`)

Reply:
Thanks. Fixed :+1: